packages/shared/src/components/utilities/DateFormat.tsx

```diff
@@ -29,7 +30,11 @@ export default function DateFormat({
   );
 
   return (
-    <time className={className} dateTime={date.toString()}>
+    <time
+      title={convertedDate.toISOString()}
```

Comment:
I would maybe keep title as toString, it's more human readable 😬

Reply:
Hmm just way longer 😀

Reply:
The shortest ones is unix timestamp 😬 :trollface: